scripts/configs.js

```diff
@@ -29,21 +31,22 @@ export const calcEnvironment = () => {
   return environment;
 };
 
-function buildConfigURL(environment) {
+function buildConfigURL(environment, root = '/') {
   const env = environment || calcEnvironment();
   let fileName = 'configs.json';
   if (env !== 'prod') {
     fileName = `configs-${env}.json`;
   }
-  const configURL = new URL(`${window.location.origin}/${fileName}`);
+  const configURL = new URL(`${window.location.origin}${root}${fileName}`);
   return configURL;
 }
 
 const getConfigForEnvironment = async (environment) => {
   const env = environment || calcEnvironment();
+  const root = getRootPath() || '/';
 
   try {
-    const configJSON = window.sessionStorage.getItem(`config:${env}`);
+    const configJSON = window.sessionStorage.getItem(`config:${env}:${root}:${root}`);
```

Comment:
```suggestion
    const configJSON = window.sessionStorage.getItem(`config:${env}:${root}:${env}`);
```

But you also need to account for if root is `/`. As in, will config b e stored at `config:/:dev` for the default or `config:dev`?

Reply:
Thanks. I typo.

Yes. Config will store as `config:dev:/` for default.